README.md

```diff
@@ -778,6 +778,7 @@ _Related: [Groupware](#groupware)_
 - [ProjectSend](https://www.projectsend.org/) - Upload files and assign them to specific clients you create. Give access to those files to your clients. ([Source Code](https://github.com/projectsend/projectsend)) `GPL-2.0` `PHP`
 - [PsiTransfer](https://github.com/psi-4ward/psitransfer) - Simple open source self-hosted file sharing solution with robust up-/download-resume and password protection. `BSD-2-Clause` `Nodejs`
 - [Pste](https://dev.pste.pw/) - Just a simple file hosting application inspired by the likes of pomf.se and teknik.io. ([Source Code](https://github.com/FoxDev/pste)) `GPL-3.0` `Python`
+- [QuickShare](https://github.com/ihexxa/quickshare) -  Quick and simple file sharing between different devices.
```

Comment:
- double space before `Quick`
- please add missing license and language/platform information
- please check the relevant boxes in the pull request template